src/app/sharedUI/items/ArtworkGridItem.tests.tsx

```diff
@@ -0,0 +1,100 @@
+import { Text } from "@artsy/palette-mobile"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { ArtworkGridItem_artwork_TestQuery } from "__generated__/ArtworkGridItem_artwork_TestQuery.graphql"
+import { __globalStoreTestUtils__ } from "app/store/GlobalStore"
+import { mockEnvironmentPayloadMaybe } from "shared/tests/mockEnvironmentPayload"
+import { renderWithWrappers } from "shared/tests/renderWithWrappers"
+import { ArtworkGridItem } from "./ArtworkGridItem"
+
+describe("ArtworkGridItem", () => {
+  const TestRenderer = () => {
+    const data = useLazyLoadQuery<ArtworkGridItem_artwork_TestQuery>(
+      graphql`
+        query ArtworkGridItem_artwork_TestQuery @relay_test_operation {
+          artwork(id: "some-id") {
+            ...ArtworkGridItem_artwork
+          }
+        }
+      `,
+      {}
+    )
+
+    return <ArtworkGridItem artwork={data.artwork!} />
+  }
+
+  it("should NOT HIDE the availabilty status dot if the 'Presantation Mode = OFF' and 'Hide Works Availability' switch = OFF", async () => {
+    const { container } = renderWithWrappers(<TestRenderer />)
```

Comment:
```suggestion
    const { UNSAFE_findByType } = renderWithWrappers(<TestRenderer />)
```

you could totally use that instead. not blocking though.